docs/reference/settings.md

```diff
@@ -412,14 +412,47 @@ Custom storage classes should subclass `django.core.files.storage.Storage`. See
 ### `WAGTAILIMAGES_EXTENSIONS`
 
 ```python
-WAGTAILIMAGES_EXTENSIONS = ['png', 'jpg']
+WAGTAILIMAGES_EXTENSIONS = ['avif', 'svg']
 ```
 
 A list of allowed image extensions that will be validated during image uploading.
 If this isn't supplied, all of AVIF, GIF, JPG, JPEG, PNG, WEBP are allowed.
 Warning: this doesn't always ensure that the uploaded file is valid as files can
 be renamed to have an extension no matter what data they contain.
 
+### `WAGTAILIMAGES_JPEG_QUALITY`
+
+```python
+WAGTAILIMAGES_JPEG_QUALITY = 75
```

Comment:
Here and below – I chose to demo the settings with realistic values, that should work on all sorts of sites, and lead to smaller images than our defaults.